File: backend/src/main/java/com/carikerja/backend/model/Employer.java
package com.carikerja.backend.model;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "employers")
public class Employer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @OneToOne
    @JoinColumn(name = "user_id", referencedColumnName = "id")
    private User user;

    private String bannerImage;
    private String logoImage;
    private String companyName;
    private String slogan;
    private String address;
    private String companySize;
    private String industry;
    private String websiteLink;
    private String instagramLink;
    private String facebookLink;
    private String linkedinLink;
    private String companyDescription;
}

File: frontend/src/types/Employer.ts
export interface Employer {
    id: number;
    userId: number;
    bannerImage: string;
    logoImage: string;
    companyName: string;
    slogan: string;
    address: string;
    companySize: 'small' | 'medium' | 'large';
    industry: string;
    websiteLink: string;
    instagramLink: string;
    facebookLink: string;
    linkedinLink: string;
    companyDescription: string;
  }